packages/http/README.md

```diff
@@ -1,152 +1,148 @@
 # Prism HTTP
 
-This package provides HTTP APIs for forwarding and mocking requests.
-
-## HTTP Forwarder
-
-> Note: The current API is still experimental - changes can be introduced in future.
-
-Module used to forward provided request. Is exposes single `forward` method and can be initialized with `IHttpOperation` and `IHttpRequest`.
+This package provides a http client featuring the ability to:
+
+- mock requests instead of hitting the server
+- validate the request and the response according to an openapi spec
+
+# Basic Usages
+
+> Note: examples in this document use the following spec
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      parameters:
+        - name: title
+          in: query
+          style: form
+          schema:
+            enum:
+              - eat
+              - drink
+      responses:
+        200:
+          description: Get Todo Items
+```
 
-### Forwarder props
+## I want to use Prism to mock responses
```

Comment:
Could we add a quick sentance on why? 

Talk about how it can let you mock without needing to hit an actual server instance, which is great for browser-based mocking, etc.